docs/superpowers/plans/2026-08-04-url-schema-hardening.md

```
@@ -0,0 +1,869 @@
+  .passthrough() // Allow extra fields for forward compatibility
+```
+
+with a plain close of the object plus a comment above the schema's closing brace:
+
+```ts
+  })
+```
+
+and add above `export const ConfigStateSchema`, appended to its existing doc comment:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)